components/authz-service/storage/postgres/migration/sql/75_update_roles_pols.up.sql

```diff
@@ -0,0 +1,78 @@
+-- replaces "infra:*" with "infra:nodes:*" and "infra:nodeManagers:*"
+-- removes incorrect action "telemetry:*"
+-- removes system permissions
+BEGIN;
+
+UPDATE iam_roles
+    SET
+        actions = '{
+            secrets:*:get,
+            secrets:*:list,
+            infra:nodes:get,
+            infra:nodes:list,
+            infra:nodeManagers:get,
+            infra:nodeManagers:list,
+            compliance:*:get,
+            compliance:*:list,
+            event:*:get,
+            event:*:list,
+            ingest:*:get,
+            ingest:*:list,
+            iam:projects:list,
+            iam:projects:get
```

Comment:
I think you missed `applications` for viewer (you have it correctly for editor below):
https://github.com/chef/automate/blob/7392f96f1e05b6009d38cd903436663d6d95c1d0/components/authz-service/storage/postgres/datamigration/sql/07_update_v2_roles.up.sql#L5